Source/Orts.Simulation/Simulation/AIs/AI.cs

```diff
@@ -418,9 +418,17 @@ private void PrerunAI(int playerTrainOriginalTrain, TTTrain.FormCommand playerTr
                 // player train is extracted from pool
                 else if (!String.IsNullOrEmpty(playerTrain.CreateFromPool))
                 {
-                    TimetablePool thisPool = Simulator.PoolHolder.Pools[playerTrain.CreateFromPool];
-                    int presentTime = Convert.ToInt32(Math.Floor(clockTime));
-                    TimetablePool.TrainFromPool extractResult = thisPool.ExtractTrain(ref playerTrain, presentTime);
+                    TimetablePool.TrainFromPool extractResult = TimetablePool.TrainFromPool.Failed;
```

Comment:
Tip for the future: you can use `var` instead of the type before a variable declaration. This is considered cleaner in C# and would mean this line is: `var extractResult = TimetablePool.TrainFromPool.Failed;`